adi_update_tools.sh

```diff
@@ -348,6 +348,11 @@ do
   elif [ $REPO = "mathworks_tools" ]
   then
     cd ./motor_control/linux_utils/
+  elif [ $REPO = "iio-oscilloscope" ]
+  then
+	rm -rf build
+	cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_COLOR_MAKEFILE=OFF -Bbuild -H.
+	cd build
```

Comment:
I'm not sure, but I think osc needs to be build in the same directory as the source tree. Have you checked if this works?

Reply:
i think if you build it locally, and run it, then it won't work;
but if you do `sudo make install` at the end, it should install things and then it should work, and things should install correctly;
that all depends if the CMakeLists.txt file is correct though

that being said, i do agree this needs testing on the target;
but if this works, i would remove the old `Makefile`; just to avoid confusion for users on EZ

Reply:
i will test it on a raspberry pi to see if there are issues with it.

Reply:
i did a build on my computer using the build folder and it worked

Reply:
did you run it successfully?
building is fine; running it is problematic

Reply:
Maybe we should remove the old Makefile.

Reply:
doing some more verification now will close for now.